modules/andes-core/broker/src/main/java/org/wso2/andes/kernel/slot/SlotManagerStandalone.java

```diff
@@ -21,15 +21,22 @@
 import com.gs.collections.impl.map.mutable.ConcurrentHashMap;
 import org.apache.commons.logging.Log;
 import org.apache.commons.logging.LogFactory;
+import org.wso2.andes.kernel.AndesException;
+import org.wso2.andes.kernel.AndesMessageMetadata;
+import org.wso2.andes.kernel.MessagingEngine;
 
+import java.util.ArrayList;
 import java.util.Iterator;
+import java.util.List;
+import java.util.Set;
 import java.util.TreeSet;
+import java.util.concurrent.ExecutionException;
 
 /**
  * This class is  responsible of slot allocating, slot creating, slot re-assigning and slot
  * managing tasks in standalone mode
  */
-public class SlotManagerStandalone {
+public class SlotManagerStandalone extends SafeDeletion {
```

Comment:
Have we extended SafeDeletion only for the common method getSafeZoneLowerBoundId?